xbmc/music/MusicDatabase.cpp

```diff
@@ -1399,6 +1400,7 @@ bool CMusicDatabase::GetArtist(int idArtist, CArtist &artist, bool fetchAll /* =
       }
     }
     m_pDS->close(); // cleanup recordset data
+    CLog::Log(LOGDEBUG, LOGDATABASE, "%s(%s) - took %d ms", __FUNCTION__, strSQL.c_str(), XbmcThreads::SystemClockMillis() - time);
```

Comment:
You can already use the typesafe fmt version here.
Like: `CLog::Log(LOGDEBUG, LOGDATABASE, "{0}({1}) - took {2} ms", __FUNCTION__, strSQL.c_str(), XbmcThreads::SystemClockMillis() - time);`

Reply:
Oh yeah of course not